Makefile

```diff
@@ -129,6 +129,16 @@ test: mocks
     done
 	go test -v --race -count 1 -tags test -coverprofile=coverage/coverage.out $$(go list ./... | grep -v 'consumers\|readers\|postgres\|internal\|opcua\|cmd')
 
+test_api:
+	@which st > /dev/null || (echo "schemathesis not found, please install it from https://github.com/schemathesis/schemathesis#getting-started" && exit 1)
+	st run api/openapi/users.yml \
+	--checks all \
+	--base-url http://localhost:9002 \
+	--header "Authorization: Bearer $(USER_TOKEN)" \
+	--contrib-openapi-formats-uuid \
+	--hypothesis-suppress-health-check=filter_too_much \
+	--stateful=links
```

Comment:
If USER_TOKEN is empty throw error

Reply:
Updated with [`0f27a04` (#2087)](https://github.com/absmach/magistrala/pull/2087/commits/0f27a04f67d4f2202a948a7c8748c13fa67ffef7)